docs/integrating-with-linters.md

```diff
@@ -29,12 +29,12 @@ The downsides of those plugins are:
 
 - You end up with a lot of red squiggly lines in your editor, which gets annoying. Prettier is supposed to make you forget about formatting – and not be in your face about it!
 - They are slower than running Prettier directly.
-- They’re yet one layer of indirection where things may break.
+- They’re yet another layer of indirection where things may break.
 
-Finally, we have tools that runs `prettier` and then immediately for example `eslint --fix` on files.
+Finally, we have tools that run `prettier` and then immediately apply `eslint --fix` on files.
 
 - [prettier-eslint](https://github.com/prettier/prettier-eslint)
 - [prettier-tslint](https://github.com/azz/prettier-tslint)
 - [prettier-stylelint](https://github.com/hugomrdias/prettier-stylelint)
 
-Those are useful if some aspect of Prettier’s output makes Prettier completely unusable to you. Then you can have for example `eslint --fix` fix that up for you. The downside is that these tools are much slower than just running Prettier.
+These are useful if some aspect of Prettier’s output makes Prettier completely unusable to you, then you can have `eslint --fix` fix that up for you. The downside is that these tools are much slower than just running Prettier.
```

Comment:
Removing "for example" isn't the right thing to do. Not all of these tools run ESLint.
Also, the comma makes the sentence strange, like it's a conditional sentence with two main clauses. "Then ..." is a separate thought here.

Reply:
Yeah you're right, the comma does make it weird. It seems like an odd phrase to me, I'll try to work with it for a bit. And I'll restore the "for example" in the previous line as well.

Reply:
The general idea of those tools is that they run extra code transformations - namely linter autofixers - **after** Prettier.